english/climatechange/climatechange.md

```diff
@@ -0,0 +1 @@
+You sit down and [accept your fate](https://en.wikipedia.org/wiki/Drowning) as the world fills with water around you.  You remember your parents, your family, your first love as the water fills your nostrils and soaks every pore in your lungs...
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.